src/memory/AmdFlash.hh

```diff
@@ -198,10 +201,11 @@ private:
 	const std::span<const SectorInfo> sectorInfo;
 	const power_of_two<size_t> sz;
 
-	static constexpr unsigned MAX_CMD_SIZE = 8;
+	static constexpr unsigned MAX_CMD_SIZE = 37;
```

Comment:
It took me a while to figure out where this '37' comes from. I only figured it out after reading the `assert(... pageSize + 5)`. Maybe it would help to write this as `5 + 32` (and `5 + 256` later)? Adding a command like `// largest command is BufferProgram` could also help.

Reply:
Will do.